src/cephadm/cephadm

```diff
@@ -2278,6 +2290,50 @@ WantedBy=ceph-{fsid}.target
     container_path=container_path,
     fsid=fsid,
     data_dir=args.data_dir)
+
+    # Type=Forking if using podman
+    else:
+        u = """# generated by cephadm
+[Unit]
+Description=Ceph %i for {fsid}
+
+# According to:
+#   http://www.freedesktop.org/wiki/Software/systemd/NetworkTarget
+# these can be removed once ceph-mon will dynamically change network
+# configuration.
+After=network-online.target local-fs.target time-sync.target
+Wants=network-online.target local-fs.target time-sync.target
+
+PartOf=ceph-{fsid}.target
+Before=ceph-{fsid}.target
+
+[Service]
+LimitNOFILE=1048576
+LimitNPROC=1048576
+EnvironmentFile=-/etc/environment
+ExecStartPre=-/usr/bin/rm -f /%t/%n-pid /%t/%n-cid
+ExecStartPre=-{container_path} rm ceph-{fsid}-%i
+ExecStart=/bin/bash {data_dir}/{fsid}/%i/unit.run
+ExecStop=-{container_path} stop ceph-{fsid}-%i
+ExecStopPost=-/bin/bash {data_dir}/{fsid}/%i/unit.poststop
+ExecStopPost=-/usr/bin/rm -f /%t/%n-pid /%t/%n-cid
+KillMode=none
+Restart=on-failure
+RestartSec=10s
+TimeoutStartSec=120
+TimeoutStopSec=120
+StartLimitInterval=30min
+StartLimitBurst=5
+Type=forking
+PIDFile=/%t/%n-pid
+
+[Install]
+WantedBy=ceph-{fsid}.target
+""".format(
+    container_path=container_path,
+    fsid=fsid,
+    data_dir=args.data_dir)
+
```

Comment:
do we really need to duplicate the unit file just for:

![image](https://user-images.githubusercontent.com/2574405/91213743-559cfa00-e712-11ea-9961-e83916de21c0.png)

?

Reply:
probably not. I had it this way because my original idea for doing it without duplication looked messy. I'll try to come up with something cleaner and change it.